frontend/homework/1/cssuse.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Selector & Box Model Demo</title>
+    <style>
+        #special-heading {
+            color: red;
+            font-weight: bold;
+        }
+        .important {
+            background-color: yellow;
+            border: 1px solid blue;
+        }
+        p {
+            margin: 10px 0;
+        }
+        a:hover {
+            text-decoration: underline;
+        }
+        h2::first-letter {
+            font-size: 2em;
+            color: green;
+        }
+
+        header h1 {
+            text-align: center;
+        }
```

Comment:
Don't target elements using tags instead use class/id names

Reply:
Noted, Can use id / classes for same.